connection.js

```diff
@@ -1369,6 +1302,35 @@ function pluralFamilyStringToSingularString(plural) { /* {{{ */
     return pluralToConvert;
 } /* }}} */
 
+function rankedUnrankedGamesString(argNameString) { /* {{{ */
+    // for the "bans" family :
+    if argNameString.includes("bans") {
```

Comment:
This should be 
```
if (argNameString.includes("bans")) {
```
right now this breaks the build.

Reply:
oh really ??
why is the () needed for a if ?
i thought it was not an obligation no ?

Reply:
Nvm this is already removed :)

Reply:
yes, i still dont understand if the problem was just an extra () after the if

Reply:
That's just the syntax for if statements in javscript. See https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Statements/if...else

> Syntax
```Js
if (condition)
   statement1
[else
   statement2]
```